### PR TITLE
Update report.md with info of how to write a logger

### PR DESCRIPTION
**Pull request by Faizan2304**

### CODE REVIEW

File: docs/report.md [hunk 1]
```diff
@@ -65,6 +64,12 @@ to load test loggers.
 > Version Note: < 15.1
 > For 15.0 version, the test loggers are also discovered from *.testadapter.dll
 
+## Write a test logger
+Follow the following step to create a test logger
+1) Take a nuget reference of package `Microsoft.TestPlatform.ObjectModel`.
+2) Implement ITestLoggerWithParameters (or ITestLogger, if your logger is not expecting any parameter). [Logger Example](https://github.com/Faizan2304/LoggerExtensions/blob/master/src/Xunit.Xml.TestLogger/XunitXmlTestLogger.cs#L19)
+3) Name your logger assemlies `*.testlogger.dll`. [Detailed]()
```
Comment (AbhitejJohn) [inline docs/report.md]: nit: assembly

File: docs/report.md [hunk 2]
```diff
@@ -65,6 +64,12 @@ to load test loggers.
 > Version Note: < 15.1
 > For 15.0 version, the test loggers are also discovered from *.testadapter.dll
 
+## Write a test logger
```
Comment (AbhitejJohn) [inline docs/report.md]: nit: Add a test logger.

File: docs/report.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ to load test loggers.
 > Version Note: < 15.1
 > For 15.0 version, the test loggers are also discovered from *.testadapter.dll
 
-## Create a test logger
+## Write a test logger
 Follow the following step to create a test logger
```
Comment (Faizan2304) [inline docs/report.md]: Fixed

File: docs/report.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ to load test loggers.
...
 1) Take a nuget reference of package `Microsoft.TestPlatform.ObjectModel`.
```
Comment (Faizan2304) [inline docs/report.md]: Fixed

File: docs/report.md (hunk 2 above)
Comment (Faizan2304) [inline docs/report.md]: Fixed

File: docs/report.md (hunk 1 above)
Comment (Faizan2304) [inline docs/report.md]: Fixed

**Pull request merged**